integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java

```diff
@@ -395,6 +396,18 @@ private void createDomainResourceAndVerify(
     DomainResource domain = createDomainResource(domainUid, domainNamespace, adminSecretName,
         TEST_IMAGES_REPO_SECRET_NAME, encryptionSecretName, replicaCount, miiImage, configMapName);
 
+    // create cluster object if not created earlier 
+    if (cluster == null) {
```

Comment:
you can also use Cluster.doesClusterExist() method instead of using class variables.